connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -203,6 +204,19 @@ message Sort {
   }
 }
 
+
+// Drop specified columns.
+message Drop {
+  // (Required) The input relation.
+  Relation input = 1;
+
+  // (Required) columns to drop.
+  //
+  // Should contain at least 1 item.
+  repeated Expression cols = 2;
```

Comment:
Does drop actually support arbitrary expressions? Shouldn't this be a repeated unresolved attribute?

Reply:
Dataset.drop takes arbitrary expressions into account https://github.com/apache/spark/blob/3b4faaf89ecd1b70d3d9b3c6096e1b70275670fb/sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala#L2952-L2957